WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -23,6 +23,23 @@
     end
   end
 
+  context "delegates" do
+    it "delegates for future comps must be delegates" do
```

Comment:
What bothers me with this phrasing is that it is tautologically true, which means it conveys no information.